Model/inverted-index.js

```diff
@@ -34,9 +27,10 @@ class Index {
       status: true,
       msg: 'Valid File',
     };
-
+    // console.log('my type', typeof file);
```

Comment:
Remove the commented out code here and every other recurrence of it throughout your code.